docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -540,6 +543,261 @@ web-2   gcr.io/google_containers/nginx-slim:0.7
 
 All the Pods in the StatefulSet are now running a new container image.
 
+### Rolling Update
+
+The `RollingUpdate` update strategy will update all Pods in a StatefuleSet, in 
+reverse ordinal order, while respecting the StatefulSet guarantees.
+
+Patch the `web` StatefulSet to apply the `RollingUpdate` update strategy.
+
+```shell
+kubectl patch statefulset web -p '{"spec":{"updateStrategy":{"type":"RollingUpdate"}}}
```

Comment:
Is the kubectl patch what we want to promote?  The advantage is that it is paste-able.  The disadvantage is that it looks like gibberish when it gets longer.

The other options are "kubectl edit" and "kubectl apply".  @pwittrock any opinion here?